core/tests/protractor/staticPagesTour.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview  End-to-end tests which visit the static pages
+                    and check for any console errors
+ */
+
+var general = require('../protractor_utils/general.js');
+
+describe('Oppia static pages tour', function() {
+  beforeEach(function() {
+    browser.get(general.SERVER_URL_PREFIX);
+  });
+
+  it('visit the splash page', function() {
```

Comment:
Before each spec, we are visiting `general.SERVER_URL_PREFIX`. So, there is nothing to do inside the spec. Should I remove it?